codespell_lib/data/dictionary.txt

```diff
@@ -3720,6 +3726,7 @@ debians->Debian's
 debth->depth
 debths->depths
 debufs->debugfs
+debugee->debuggee
```

Comment:
I'm not sure this is a real word, I assume you mean the person who did the debugging?

Reply:
e.g. check https://en.wiktionary.org/wiki/debuggee